### ISSUE TITLE
🐛 [Bug Report] [Quest 9853] Gurok the Usurper can not complete

### ISSUE THREAD

**Issue by betayi** [cmangos/issues#3244]
### Bug Details

the quest item used at Earthen Brand in Nagrand can not summon the target creature Gurok the Usurper

### Steps to Reproduce

1. .q a 9853 
2. .add 24502 7
3. .go obj 22661 
4. then use item 24502,player cast spell 32036 for 10 second to summon Gurok the Usurper ,but faild at all

### Expected behavior

it should not failed and Gurok the Usurper should be summoned

### Core SHA1 Commit Hash

e3ae09278b57ebbbc149a496345e562864075ee2

### Database SHA1 Commit Hash

6be988dae2a085c744be763192ef132e8255a3a0

### Operating System

unbuntu20.04

### Client Version

2.4.3 (The Burning Crusade)

**Comment by AnonXS** [comment]
Sometimes shows "spell failed" in castbar text so probably related to spell.

```
.s 32036

.q a 9853
.add 24502 7
.go o 22661
```

**Comment by LEO33SoFo** [comment]
Issue still happening as of https://github.com/cmangos/mangos-tbc/commit/1c82f5b3e419b3aa01563e8b7f5a8f82ab1c3237 with https://github.com/cmangos/tbc-db/commit/2f994e648a3be68b5c6fdd8e62962547ca69e757.

**Comment by f0n1x** [comment]
Still happening as of https://github.com/cmangos/mangos-tbc/commit/46ce862172aee1d1ff969f33e20374c3c869d40b and https://github.com/cmangos/tbc-db/commit/7cbedc9d1fa1139f5495bfdc5e4a0b76a719c8bb

Took a video:
https://youtu.be/Hrw1gM8efCU

**Comment by f0n1x** [comment]
Some digging... I've only provided values which seem to be relevant, you may run the corresponding sql searches on the world database to get complete results. I've omitted a lot of default values.

SELECT * FROM `item_template` WHERE entry = 24502;
class = 15 --Miscellaneous
name = "Warmaul Skull" --name of the item
displayid = 37238 --modelID of the item
Flags = 1088 --(1024 is ITEM_FLAG_USES_RESOURCES, 64 is ITEM_FLAG_PLAYERCAST)
stackable = 7 --Stacks in lots of 7
spellid_1 = 32036 --spell that is cast "Summon Gurok"
bonding = 4 --quest item

SELECT * FROM `spell_template` WHERE id = 32036;
RequiresSpellFocus = 1389 --Need to be close to GO 1389, which is "Earthen Brand"
CastingTimeIndex = 7 --10,000 milisecords, aka 10 sec cast
InterruptFlags= 15 --8 is SPELL_INTERRUPT_FLAG_AUTOATTACK, 4 is SPELL_INTERRUPT_FLAG_INTERRUPT, 2 is SPELL_INTERRUPT_FLAG_DAMAGE, 1 is SPELL_INTERRUPT_FLAG_MOVEMENT
ProcChance = 101 --101%?
RangeIndex = 1 --SELF_ONLY
Reagent1 = 24502 --Warmaul Skull (items consumed when cast?)
ReagentCount1 = 6 --Number consumed?
EffectMiscValue1 = 11420 --No idea here
SpellVisual = 1523 --Visual effect when spell is cast?
SpellIconID = 2289 --Icon ID for spell being cast?
SpellName = Summon Gurok --Text that appears when casting the spell
SchoolMask = 1 --SPELL_SCHOOL_MASK_NORMAL
IsServerSide = 0 --Default Spell.sql Data

SELECT * FROM `gameobject_template` WHERE data0 = 1389;
type = 8 --GAMEOBJECT_TYPE_SPELL_FOCUS
displayID = 5812 --Graphic model id sent to the client from GameObjectDisplayInfo.dbc.
name = Earthen Brand --Name of the GO
size = 2 --base size of the GO texture
data0 = 1389 --spellFocusType (from SpellFocusObject.dbc)
data1 = 10 --diameter (so radius*2)

Basically everything seems to look correct. I'm a bit confused on the EffectMiscValue from the Spell, but everything seems to line up. Not sure why Gurok isn't being summoned...

**Comment by f0n1x** [comment]
Adding more info for posterity:
@insunaa has a fix that works for the wotlk core:
UPDATE spell_template SET EffectImplicitTargetA1=1 WHERE id=32036;
it doesn't seem to resolve the issue on the tbc core though.

**Comment by insunaa** [comment]
Another issue here is that the skulls which are currently hard-spawned should instead be spawned upon receiving that event.
I believe https://www.wowhead.com/wotlk/npc=18196/gurok-event-controller should be responsible for this thing

**Comment by betayi** [comment]
Resolved by [https://github.com/cmangos/tbc-db/commit/abda7d80e455d05067aae6187ff118428302aa7a](url)

**Comment by miraco** [comment]
Are you sure they got fixed? The changes I uploaded are only for the spawnings, spell (iirc) still failed and needs a proper fix

**Comment by cpevors** [comment]
I agree with @miraco -- we fixed half of the issue, but my spell continues to 'fail' and I'm not able to complete this quest.

**Comment by al3xc1985** [comment]
@betayi reopen the issue pls
